src/test/java/gov/healthit/chpl/aqa/stepDefinitions/BaseSteps.java

```diff
@@ -63,4 +66,15 @@ public WebDriverWait getWait() {
     public void setWait(final WebDriverWait wait) {
         this.wait = wait;
     }
+
+    /**
+     * Returns element holding main content.
+     * @param driver WebDriver
+     * @return element holding main content
+     */
+    public static WebElement mainContent(final WebDriver driver) {
```

Comment:
I like this, but it's in the wrong file. It should be in a new "BasePage.java" file in the pages directory, not in the steps file